code/__DEFINES/traits.dm

```diff
@@ -615,6 +615,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define DRONE_SHY_TRAIT "drone_shy"
 /// Pacifism trait given by stabilized light pink extracts.
 #define STABILIZED_LIGHT_PINK_TRAIT "stabilized_light_pink"
+/// Climbable trait, given and taken by the climbable element
+#define TRAIT_CLIMBABLE "trait_climbable"
```

Comment:
wtf why are you putting a trait in trait sources

Reply:
Relocate this and also give it a better description/documentation.
By reading this alone I have zero idea of what the trait does, besides that it's related to some element of the same name.

Reply:
Don't forget about this one, please.